### ISSUE TITLE
Admin Assistants (AAs) need read-only access to notifications and data for people in other departments

### ISSUE THREAD

**Issue by kevinreiss** [pulibrary/approvals#91]
As a departmental administrator I need to know who will/will not be working in my department's physical space even if they are not technically in my department so that I can manage space use.

Example use case: staff who administratively part of RBSC but who physically reside at 693. Departmental administrators at 693 need to know about the leave status and locations of these users.

Maybe once we have the calendar report it will satisfy this use case, and they don't actually need the email notifications.

Might require filtering by a list of individuals, or by physical location (can we get that data?)

**Comment by sdellis** [comment]
In simpler terms, this is simply saying that there can be more than one proxy for a department manager, correct @kevinreiss ?

**Comment by kevinreiss** [comment]
Handled via delegation.
